src/Screen.js

```diff
@@ -1,5 +1,6 @@
-import React, { Component } from 'react';
-import styled from 'styled-components'
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
```

Comment:
having it as `import { func, string, ...etc } from 'prop-types';` can save you some time